src/nodeinfoprovider/nodeinfoprovider.hpp

```diff
@@ -45,6 +45,7 @@ class NodeInfoProvider : public aos::iam::nodeinfoprovider::NodeInfoProviderItf
     aos::Error StoreNodeStatus(const aos::NodeStatus& status) const;
     aos::Error ReadNodeStatus(aos::NodeStatus& status) const;
 
+    static constexpr auto cNodeStatusLen         = 16;
```

Comment:
As we agreed before, put node info provider implementation under aos::zephyr::nodeinfoprovider namespace.

Reply:
Done in the next commit